__tests__/page.test.tsx

```diff
@@ -0,0 +1,21 @@
+import {describe, expect, test} from '@jest/globals';
+import Home from "../src/app/page";
+import "@testing-library/jest-dom";
+import { fireEvent, render, screen } from "@testing-library/react";
+import { renderHook } from '@testing-library/react-hooks';
+
+describe("Calculator", () => {
```

Comment:
The final test should be just simple to confirm overall testing can be made further 
```
import {  render, screen } from "@testing-library/react";

describe("Calculator", () => {
    it('should pass all the tests', () => {
    render(<h1>Hello world!</h1>)

    expect(screen.getByText("Hello world!")).toBeInTheDocument();
    });
});
```

Here are some key notes for you to acknowledge:
- use `render` from `@testing-library/react`, `renderHook` is for rendering hooks (you had a component)
- `import {describe, expect, test} from '@jest/globals';` is not needeed because there are mounter globally on its own
- `import "@testing-library/jest-dom";` - you've already set it up in the `jest.setup.ts` file. No need to do it twice
- `fireEvent` was not used, so we remove it
- `expect(screen.findAllByTestId("result")).toBeInTheDocument` - `toBeInTheDocument` in a function, it should be called like this `toBeInTheDocument()`, otherwise you do not check anything